isisd/isis_zebra.c

```diff
@@ -699,6 +699,20 @@ static int isis_opaque_msg_handler(ZAPI_CALLBACK_ARGS)
 	return ret;
 }
 
+static int isis_zebra_client_close_notify(ZAPI_CALLBACK_ARGS)
+{
+	int ret = 0;
+
+	struct zapi_client_close_info info;
+
+	if (zapi_client_close_notify_decode(zclient->ibuf, &info) < 0)
+		return -1;
+
+	isis_ldp_sync_handle_client_close(&info);
```

Comment:
As ZAPI_CALLBACK_ARGS is having the vrf_id 
Can it be passed from here are used in isis_ldp_sync_handle_client_close() ?

Reply:
@KaushikNiral Actually, zebra is coded to send the ZEBRA_CLIENT_CLOSE_NOTIFY with VRF_DEFAULT (since the client close event is not associated with a specific vrf).

Since the vrf_id will always be VRF_DEFAULT, I'd recommend that we leave the parameters to isis_ldp_sync_handle_client_close() as is.

Thanks, Karen

Reply:
Okay, got it Karen 

Thanks, 
Kaushik